src/pages/registration/contributors_tab/components/AddContributorTableRow.tsx

```diff
@@ -24,31 +23,10 @@ export const CristinPersonTableRow = ({
   const activeAffiliations = filterActiveAffiliations(cristinPerson.affiliations);
   const personIsSelected = cristinPerson.id === selectedPerson?.id;
 
-  const selectedPersonHasAllAffiliations = !activeAffiliations.some(
-    (affiliation) => !selectedPerson?.affiliations.some((a) => a.organization === affiliation.organization)
-  );
-
-  const hasSelectedAll = personIsSelected && selectedPersonHasAllAffiliations;
-
   const resetPersonSelection = () => setSelectedPerson(undefined);
 
   return (
     <TableRow selected={personIsSelected}>
-      <TableCell>
-        <IconButton
-          data-testid={dataTestId.registrationWizard.contributors.selectEverythingForContributor}
-          onClick={() => {
-            if (hasSelectedAll) {
-              resetPersonSelection();
-            } else {
-              setSelectedPerson({ ...cristinPerson, affiliations: activeAffiliations });
-            }
-          }}
-          color="primary"
-          title={t('registration.contributors.select_all')}>
-          {hasSelectedAll ? <CheckCircle color="info" fontSize="large" /> : <ControlPointIcon fontSize="large" />}
-        </IconButton>
-      </TableCell>
       <TableCell>
         <Box sx={{ display: 'flex', gap: '0.25rem', alignItems: 'center' }}>
           <IconButton
```

Comment:
Kunne kanskje droppa `size="small"` for denne nå, da det nå vil være knappen vi vil at de skal trykke på (som tidligere var "velg alt"). Opp til deg